### PR TITLE
Add CD for Staging MainNet

### PR DESCRIPTION
**Pull request by juan-rsk**
Remember to set the secrets before to run the action.
The action is run with every PR merged and closed to 'main'... if you now prefer other deploy strategy, just let me know before merging.
Thanks.

### CODE REVIEW

File: .github/workflows/deploy_staging_MainNet_UI.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Build site
+        run: >
+          VUE_APP_COIN='main'
+          VUE_APP_MANIFEST_EMAIL='user@rsk.co'
+          VUE_APP_MANIFEST_APP_URL='https://2wp-app.rsk.co'
+          VUE_APP_API_BASE_URL='https://2wp-api.rsk.co'
```
Comment (ronaldsg20) [inline .github/workflows/deploy_staging_MainNet_UI.yml:35]: I think we have to point to staging-mainet deployed `API` url

File: .github/workflows/deploy_staging_TestNet_UI.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Build site
+        run: >
+          VUE_APP_COIN='test'
+          VUE_APP_MANIFEST_EMAIL='user@rsk.co'
+          VUE_APP_MANIFEST_APP_URL='https://2wp-app.testnet.rsk.co'
+          VUE_APP_API_BASE_URL='https://2wp-api.testnet.rsk.co'
```
Comment (ronaldsg20) [inline .github/workflows/deploy_staging_TestNet_UI.yml:35]: Same observation here, point to staging-testnet `API` url

**Review by ronaldsg20 — CHANGES_REQUESTED** [review]

File: .github/workflows/deploy_staging_MainNet_UI.yml (hunk 1 above)
Comment (juan-rsk) [reply .github/workflows/deploy_staging_MainNet_UI.yml]: You are right.
Thanks.
Fixed.

File: .github/workflows/deploy_staging_TestNet_UI.yml (hunk 2 above)
Comment (juan-rsk) [reply .github/workflows/deploy_staging_TestNet_UI.yml]: You are right.
Thanks.
Fixed.

**Review by ronaldsg20 — APPROVED** [review]

**Pull request merged**